src/Sulu/Bundle/MediaBundle/Media/FormatManager/FormatManager.php

```diff
@@ -452,16 +456,18 @@ protected function getFile($uri, $mimeType)
     /**
      * create a local temp file for the original.
      *
-     * @param $content
+     * @param resource $resource
      *
      * @return string
      */
-    protected function createTmpFile($content)
+    protected function createTmpFile($resource)
     {
-        $tempFile = tempnam(sys_get_temp_dir(), 'media_original');
+        $tempFile = tempnam(null, 'media_original');
```

Comment:
there was problem with the timpfile at early beginning with the media bundle and since then I changed to use the tempnam